package-lock.json

```diff
@@ -1,5823 +1,7 @@
 {
     "name": "spryker-sdk",
-    "lockfileVersion": 2,
```

Comment:
LockfileVersion is changed. Did you install js dependencies inside the docker container?